src/components/Calculator.js

```diff
@@ -33,96 +28,77 @@ class Calculator extends Component {
-        >
+        <button className="sign" type="button" value="÷" onClick={clickEvent}>
           ÷
         </button>
-        <button type="button" value="7" onClick={this.clickEvent}>
+        <button type="button" value="7" onClick={clickEvent}>
           7
         </button>
-        <button type="button" value="8" onClick={this.clickEvent}>
+        <button type="button" value="8" onClick={clickEvent}>
           8
         </button>
-        <button type="button" value="9" onClick={this.clickEvent}>
+        <button type="button" value="9" onClick={clickEvent}>
           9
         </button>
-        <button
-          className="sign"
-          type="button"
-          value="x"
-          onClick={this.clickEvent}
-        >
+        <button className="sign" type="button" value="x" onClick={clickEvent}>
           x
         </button>
-        <button type="button" value="4" onClick={this.clickEvent}>
+        <button type="button" value="4" onClick={clickEvent}>
           4
         </button>
-        <button type="button" value="5" onClick={this.clickEvent}>
+        <button type="button" value="5" onClick={clickEvent}>
           5
         </button>
-        <button type="button" value="6" onClick={this.clickEvent}>
+        <button type="button" value="6" onClick={clickEvent}>
           6
         </button>
-        <button
-          className="sign"
-          type="button"
-          value="-"
-          onClick={this.clickEvent}
-        >
+        <button className="sign" type="button" value="-" onClick={clickEvent}>
           -
         </button>
-        <button type="button" value="1" onClick={this.clickEvent}>
+        <button type="button" value="1" onClick={clickEvent}>
           1
         </button>
-        <button type="button" value="2" onClick={this.clickEvent}>
+        <button type="button" value="2" onClick={clickEvent}>
           2
         </button>
-        <button type="button" value="3" onClick={this.clickEvent}>
+        <button type="button" value="3" onClick={clickEvent}>
           3
         </button>
-        <button
-          className="sign"
-          type="button"
-          value="+"
-          onClick={this.clickEvent}
-        >
+        <button className="sign" type="button" value="+" onClick={clickEvent}>
           +
         </button>
         <button
           className="number zero"
           type="button"
           value="0"
-          onClick={this.clickEvent}
+          onClick={clickEvent}
         >
           0
         </button>
-        <button type="button" value="." onClick={this.clickEvent}>
+        <button type="button" value="." onClick={clickEvent}>
           .
         </button>
         <button
           className="assign"
           type="button"
           value="="
-          onClick={this.clickEvent}
+          onClick={clickEvent}
+          data-testid="answer"
         >
           =
         </button>
       </div>
-    );
-  }
-}
+    </div>
```

Comment:
- [ ] @noelfoka Great work on your calculator page, however there is a small error in placement of the title, check the comparison below for better understanding:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/noelfoka/math-magicians/assets/61408860/9adefda8-1d90-486d-a653-2a7e11fadbf4) |  ![image](https://github.com/noelfoka/math-magicians/assets/61408860/b6dd0d40-6dc9-4cb6-8b6d-fd2f78d666b3)